src/main/java/UseCase/playSpaceInteractor.java

```diff
@@ -15,21 +17,56 @@ public class playSpaceInteractor implements playSpaceInputBoundary {
 
     // instance of global playlistLibrary
     private final Database.songLibrary songLibrary;
+    private final OutputBoundary.spacePlayedOutputBoundary spacePlayedOutputBoundary;
+    private final Object sync;
+    private final InputData.playSpaceInputData playSpaceInputData;
+    private final songOutputBoundary songOutputBoundary;
+    private boolean keepPlaying;
 
     /**
      * constructor
      */
-    public playSpaceInteractor(){
+    public playSpaceInteractor(OutputBoundary.spacePlayedOutputBoundary spacePlayedOutputBoundary,
+                               playSpaceInputData playSpaceInputData,
+                               songOutputBoundary songOutputBoundary){
+        this.spacePlayedOutputBoundary = spacePlayedOutputBoundary;
         this.songLibrary = Database.songLibrary.getInstance();
+        this.sync = MusicPlayer.getInstance().getSync();
+        this.keepPlaying = true;
+        this.playSpaceInputData = playSpaceInputData;
+        this.songOutputBoundary = songOutputBoundary;
     }
 
     /**
      * calls playSong's actuallyPlaySong function
      */
     @Override
-    public void playSpace(songInputData songInputData){
-        // get values
-        Song songToPlay = songInputData.getSong();
+    public void playSpace(playSpaceInputData playSpaceInputData){
```

Comment:
Just a reminder that we still need to ask/decide whether input datas will be given on instantiation or on method call. (aka do we need to make a new interactor for every use case call?) Maybe add this to the pull notes.